docs/power.rst

```diff
@@ -53,12 +53,18 @@ Functions
     button is pressed (which resets the board) or, in battery power,
     when the USB cable is inserted.
 
+    When the ``run_every`` parameter is set to ``True`` (the default), any
+    function scheduled with :py:meth:`microbit.run_every<microbit.run_every>`
+    will still run while the board sleeps. When the scheduled time is reached
+    the micro:bit will momentarily wake up to run the scheduled function
+    and then automatically go back to sleep.
```

Comment:
The second sentence here is more of an implementation detail, so I suggest making that clear, and giving more detail.  Eg:
```
Implementation note: To make sure the `run_every`'s continue to run, the
micro:bit will wake up from deep sleep at the correct time to run the next
scheduled `run_every`, and then go back to deep sleep mode as soon as
that `run_every` completes.  It will continue to do this until the deep sleep
finishes due to the `ms` timeout being reached, or a `wake_on` event
occurs.
```

Reply:
I think you are right, that paragraph probably contained a bit too much detail, so I tried to simplify it a bit (it says almost the same, but hopefully it's less technical), and then added the technical detail to the "Detailed Information" section in https://github.com/bbcmicrobit/micropython/pull/769/commits/18dd86d95a04e644cb0aa43beb4cba11acb942dc